config_defaults/subtests/docker_cli/run.ini

```diff
@@ -54,6 +54,19 @@ last_name_sticks = yes
 names_count = 1000
 exit_status = 0
 
+[docker_cli/run/run_tmpfs]
```

Comment:
Don't forget to add ``run_tmpfs`` to the ``subsubtests`` CSV list here: https://github.com/afomm/autotest-docker/blob/afomdev/config_defaults/subtests/docker_cli/run.ini#L2